### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Attumm**
Updated Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,78 @@
...
+    >>> 'foo' in r_dic
+    False
+    >>> r_dic['foo'] = 4
+    >>> r_dict['foo']
+    4
+    >>>
+```
+
+### Note
 This project is used in different companies in production.
```
Comment (techhazard) [inline README.md:80]: Suggestion:

> This project is used by different companies in production.

**Review by techhazard — CHANGES_REQUESTED** [review]
Hi @Attumm,

I've found redis-dict very helpful so far, so I thought I'd contribute a little.

I made some suggestions regarding your new documentation. I hope you can appreciate them.

**Comment by Attumm** [comment]
Thanks @techhazard, great suggestions.

**Review by Attumm — COMMENTED** [review]
thanks to @techhazard

**Comment by techhazard** [comment]
You're welcome, @Attumm :smiley:

**Pull request merged**